tools/transformer/src/transformations/tagExperimental.js

```diff
@@ -0,0 +1,26 @@
+const stableOperationIds = require("../operations.stable.json").stableIds;
+
+/**
+ * Transforms the path methods to mark them as "experimental" where paths are not flagged as stable.
+ *
+ * @param {*} api OpenAPI JSON File
+ * @returns OpenAPI JSON File
+ */
+function applyAddExperimentalTag(api) {
+  Object.values(api.paths).forEach((path) => {
+    Object.values(path).forEach((requestMethod) => {
+      if (
+        requestMethod.operationId &&
+        !stableOperationIds.includes(requestMethod.operationId)
+      ) {
+        requestMethod.description = `${requestMethod.description}`;
+        requestMethod["x-experimental"] = true;
```

Comment:
not blocking, should this be x-xgen-experimental?

Reply:
Agreed, updated